ArduCopter/commands_logic.cpp

```diff
@@ -160,6 +160,10 @@ bool Copter::start_command(const AP_Mission::Mission_Command& cmd)
         break;
 #endif
 
+    case MAV_CMD_DO_WINCH:                             // Mission command to control winch
```

Comment:
Shouldn't we add this to the verify_command list? It will send a warning message if it isn't there.

Reply:
added MAV_CMD_DO_WINCH to the verify_command functions too, thanks.